bids/variables/tests/test_entities.py

```diff
@@ -73,7 +73,7 @@ def test_get_collections_merged(layout1):
     assert len(collection.variables) == 8
     vals = collection.variables['RT'].values
     ents = collection.variables['RT'].index
-    assert len(ents) == len(vals) == 4096
+    assert len(ents) == len(vals) == 4181
```

Comment:
Why is this changing?

Reply:
Ah, forgot to flag this. Not sure why I had to change this, but the test failed otherwise. I added some blank files to the test dataset, so I assumed it was because of that, but I wasn't sure what the test was actually doing.

Reply:
I'm looking into this. I think something's going wrong, since `collection.to_df()` still has length 4096.

Reply:
This happens because you're adding an extra BOLD file with no run entity. I would remove that extra file; your tests don't need it and it's at least against the spirit of BIDS to mix files with and without run and no other distinguishing entities.

https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#run

Reply:
Great, I've made the change!